pkg/messages/api_errors.go

```diff
@@ -88,4 +88,15 @@ const (
 	ErrLockOwnerEmpty             = "LockOwner is empty in lock store %s"
 	ErrExpiryInSecondsNotPositive = "ExpiryInSeconds is not positive in lock store %s"
 	ErrLockStoreNotFound          = "lock store %s not found"
+
+	// Workflow
+	ErrWorkflowNameMissing    = "workflow name is not configured"
+	ErrMissingOrEmptyInstance = "no instance or empty instance was provided"
```

Comment:
For constants in this block, they should all have `Workflow` in the name. For example, this one should be `ErrMissingOrEmptyWorkflowInstance` or something like that